docs/en/getting-started/for_distributed.md

```diff
@@ -147,33 +147,33 @@ redis://tom:mypassword@myjfs-sh-abc.apse1.cache.amazonaws.com:6379/1    /mnt/myj
 ```
 
 :::note
-By default, CentOS 6 does not mount the network file system at boot time, you need to run the command to enable automatic mounting support for the network file system: `sudo chkconfig --add netfs`
+By default, CentOS 6 does not mount the network file system during boot, you need to run the command `sudo chkconfig --add netfs` to enable automatically mounting.
 :::
 
 ### 6. Unmounting the file system
 
-You can unmount the JuiceFS file system (assuming the mount point path is `mnt`) with the `juicefs umount` command.
+You can unmount the JuiceFS file system (assuming the mount point path is `mnt`) by the command `juicefs umount`.
 
 ```shell
 juicefs umount mnt
 ```
 
 #### Unmounting failure
 
-If the command fails to unmount the file system after execution, the prompt is `Device or resource busy`.
+If the command fails to unmount the file system after execution, it will prompt `Device or resource busy`.
 
 ```shell
 2021-05-09 22:42:55.757097 I | fusermount: failed to unmount mnt: Device or resource busy
 exit status 1
 ```
 
-This may happen because some programs are reading and writing files in the file system. To ensure data security, you should first troubleshoot which programs are interacting with files on the file system (e.g. via the `lsof` command) and try to end the interaction between them before re-executing the unmount command.
+This failure happens probrobly because some programs are reading and writing files in the file system when the unmount command is executed. To ensure data security, you should first troubleshoot which programs are interacting with files on the file system (e.g. via the command `lsof`) and try to terminate the interaction between them before re-executing the unmount command.
```

Comment:
This failure happens probably because some programs are reading or writing files in the file system when executing `unmount` command. To avoid data loss, you should first find out which processes are accessing files in the file system (e.g. via the command `lsof`) and try to release the files before re-executing the `unmount` command.